__tests__/src/components/PackageListItem.test.js

```diff
@@ -2,43 +2,39 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import expect from 'expect';
 
-import PackageListItem from '../../../src/components/PackageListItem';
+import PackageListItem from '../../../src/components/common/PackageListItem';
 
 describe("<PackageListItem />", () => {
   it("render one panel item", () => {
     const wrapper = shallow(<PackageListItem name="header"
-                                             packageUrl="core/demo-package" />);
-    expect(wrapper.find('div').length).toBe(4);
+                                             title="title"
+                                             publisherName="publisher"/>);
+    expect(wrapper.find('div').length).toBe(3);
```

Comment:
It would be much more informaive and assuring to assert the full html contents of rendered component, rather than div count, which tells almost nothing.